aaronsekisambu/my-personal-projects
# Desigin tear wireframes

## Description
- Scaling out a wireframe of a simple design magazine website

## How to Manually Test
- Clone the repo and fetch all branches from the origin
- Check to `target-tears`
- Run the index.html file
## Screenshots
